packages/web/src/components/form-fields/PriceField.tsx

```diff
@@ -24,12 +23,12 @@ const messages = {
 export const PriceField = (props: TextFieldProps) => {
   const [{ value }, , { setValue: setPrice }] = useField<number>(props.name)
   const [humanizedValue, setHumanizedValue] = useState(
-    value ? toHumanReadable(value) : null
+    value ? decimalIntegerToHumanReadable(value) : null
   )
 
   useEffect(() => {
     if (humanizedValue !== null) {
-      const dehumanizedValue = fromHumanReadable(humanizedValue)
+      const dehumanizedValue = decimalIntegerFromHumanReadable(humanizedValue)
```

Comment:
nit: this name is lols. wondering if there's a better way to do this. i don't suppose Formik has a human-readable number input of some sort?
just calling out but i don't have a better solution to propose, so feel free to ignore.